remote-trait-object-macro/src/service.rs

```diff
@@ -43,32 +43,42 @@ impl<T: Parse> Parse for SingleArg<T> {
 #[derive(Default)]
 struct MacroArgsRaw {
     pub serde_format: Option<syn::Path>,
-    pub remote_only: Option<syn::LitBool>,
+    pub no_stub: Option<()>,
+    pub no_skeleton: Option<()>,
 }
 
 struct MacroArgs {
     pub serde_format: syn::Path,
-    pub remote_only: bool,
+    pub no_stub: bool,
+    pub no_skeleton: bool,
 }
 
 impl MacroArgsRaw {
     fn update(&mut self, ts: TokenStream2) -> syn::parse::Result<()> {
-        let x: SingleArg<TokenStream2> = syn::parse2(ts.clone())?;
+        if let Ok(x) = syn::parse2::<syn::Ident>(ts.clone()) {
+            if x == quote::format_ident!("no_stub") {
```

Comment:
The variable name `x` is used in many places. What is the meaning that the `x` indicates?

Reply:
I gave a meaningful name